src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

```diff
@@ -19,8 +18,9 @@ describe('Navigate to Message Details ', () => {
       .eq(0)
       .should('contain', 'Print');
     cy.realPress('Tab');
-    cy.get(Locators.BUTTONS.BUTTON_MOVE).should('have.focus');
+    cy.get(Locators.BUTTONS.BUTTON_MOVE).should('be.visible');
```

Comment:
@fazilqa if we are tabbing to a specific element, shouldn't we validate that the element actually gets focus, like it was prior to this change?

Reply:
refactored